custom/icds_reports/utils/aggregation_helpers/distributed/agg_awc.py

```diff
@@ -69,18 +69,18 @@ def aggregation_query(self):
             1,
             'no',
             5,
-            0,
-            0,
-            0,
-            0,
-            0
```

Comment:
@emord/ @calellowitz  this is on me. 
I fixed it in the next commit on same PR.
Also I verified its affecting dashboard as these values(num_launched_block,num_launched_district,num_launched_state) are not used at this level(awc level)

Reply:
cross checked. cbe and vhsnd is used in sdd.